modules/gapi/src/backends/fluid/gfluidcore_func.simd.hpp

```diff
@@ -1450,6 +1468,38 @@ MULC_SIMD(float, float)
 
 #undef MULC_SIMD
 
+//-------------------------
+//
+// Fluid kernels: AbsDiffC
+//
+//-------------------------
+
+#define ABSDIFFC_SIMD(SRC)                                                          \
+int absdiffc_simd(const SRC in[], const float scalar[], SRC out[],                  \
```

Comment:
always_inline ?

Reply:
Because of dynamic dispatching support, any key words before return type break compilation.